source/fundamentals/connection.txt

```diff
@@ -126,6 +126,37 @@ replica set named "myRs".
 
    mongodb://host1:27017,host2:27017,host3:27017
 
+.. _mongo-client-settings-multiple-hosts:
+
+The code snippets below show how to specify multiple hosts to a ``MongoClient``
+instance using either the ``ConnectionString`` or ``MongoClientSettings`` class.
```

Comment:
suggestion: conformity

```suggestion
The following examples show how to specify multiple hosts to a ``MongoClient``
instance using either the ``ConnectionString`` or ``MongoClientSettings`` class.
```

Reply:
This phrasing (code snippet) was obtained from the [authentication page](https://github.com/mongodb/docs-java/blob/904bef566435a2e8b93cdc29666829b9671a61bb/source/fundamentals/auth.txt#L60). I am not attached to either phrasing. Perhaps it would make sense to make a decision as a team on what whether "code snippet" or "example" should be used?

I agree that "below" should be changed to "the following" as per [style guide](https://github.com/mongodb/docs-java/blob/904bef566435a2e8b93cdc29666829b9671a61bb/source/fundamentals/auth.txt#L60), great catch. Going forward, I will try to avoid using "above", "below" when referencing examples.

Committing suggested change.